src/main/java/com/minecolonies/coremod/generation/defaults/DefaultResearchProvider.java

```diff
@@ -75,7 +76,9 @@ public Collection<ResearchEffect> getResearchEffectCollection()
         effects.add(new ResearchEffect(BLOCK_ATTACKS).setTranslatedName("Knight Shield Blocking Chance +%3$s%%").setLevels(new double[] {0.05, 0.1, 0.25, 0.5}));
         effects.add(new ResearchEffect(BLOCK_BREAK_SPEED).setTranslatedName("Citizen Block Break Speed +%3$s%%").setLevels(new double[] {0.1, 0.25, 0.5, 1, 2}));
         effects.add(new ResearchEffect(BLOCK_PLACE_SPEED).setTranslatedName("Citizen Block Place Speed +%3$s%%").setLevels(new double[] {0.1, 0.25, 0.5, 1, 2}));
-        effects.add(new ResearchEffect(CITIZEN_CAP).setTranslatedName("Increase Max Citizens +%s").setLevels(new double[] {25, 75, 125, 1000}));
+        effects.add(new ResearchEffect(CITIZEN_CAP).setTranslatedName("Increase Max Citizens +%s")
+                      .setLevels(new double[] {CitizenConstants.CITIZEN_LIMIT_OUTPOST, CitizenConstants.CITIZEN_LIMIT_HAMLET, CitizenConstants.CITIZEN_LIMIT_VILLAGE,
```

Comment:
You need to also `runData` if you want this change to actually do anything.  (Otherwise your other changes will reduce the citizen cap.)

Also the `+%s` part is no longer correct with this change, you should change it to `Increase Max Citizens to %s`.